### ISSUE TITLE
macOS Blank Frame

### ISSUE THREAD

**Issue by lancelet** [nix-community/emacs-overlay#21]
I'm placing a link here to the report of a problem with emacsGit that has been reported upstream.
https://debbugs.gnu.org/cgi/bugreport.cgi?bug=39883

(I am not the reporter on the GNU bug tracker - thank you to that person!)

Based on the discussions on the GNU bug tracker, this seems to be a Nix-specific issue. It's not clear (at the time I'm posting this) whether it will be resolved upstream, or whether it's something that the Nix macOS ecosystem will have to address.

**Comment by wahjava** [comment]
This problem should be fixed by emacs-mirror/emacs@c019f03e38850b3ba3a2bcbcc84960d5b8d9663f so latest version should build just fine, although you do need to override `CFLAGS` attribute of `emacs` derivation to include `"-DMAC_OS_X_VERSION_MAX_ALLOWED=101500"` (on macOS Catalina, at least) but since I'm not well informed if it is going to be compatible with older versions of macOS, I am not submitting a PR:

```diff
diff --git a/default.nix b/default.nix
index b8ac813..a11d650 100644
--- a/default.nix
+++ b/default.nix
@@ -34,6 +34,7 @@ let
       ./patches/tramp-detect-wrapped-gvfsd.patch
       ./patches/clean-env.patch
     ];
+    CFLAGS = "-DMAC_OS_X_VERSION_MAX_ALLOWED=101500";
     postPatch = ''
        substituteInPlace lisp/loadup.el \
          --replace '(emacs-repository-get-version)' '"${repoMeta.rev}"' \
```

HTH

**Comment by adisbladis** [comment]
This report is really old and according to @wahjava it should already be fixed.
If not please reopen.
